pkg/devspace/helm/v3/client.go

```diff
@@ -76,6 +76,7 @@ func (c *client) InstallChart(ctx devspacecontext.Context, releaseName string, r
 		var chartRepo string
 		chartName, chartRepo = generic.ChartNameAndRepo(helmConfig)
 		args = append(args, chartName)
+		args = append(args, "--create-namespace")
```

Comment:
Problem is we cannot do that as `--create-namespace` needs namespace permissions which in spaces are not given. Also this would only solve the case for helm install and not kubectl deployments